_includes/home.html

```diff
@@ -1,4 +1 @@
-{% assign home = "" %}
```

Comment:
I have tested it with case 3. It works fine there too.
https://mandarjog.github.io/docs/tasks/metrics-logs.html

look for `https://raw.githubusercontent.com/istio/istio/release-0.1/samples/apps/bookinfo/mixer-rule-additional-telemetry.yaml` link there